content/learning-paths/servers-and-cloud-computing/cca-container/cca-container.md

```diff
@@ -135,7 +136,61 @@ buildroot login:
 ```
```

Comment:
In order to be consistent with the previous example, I would suggest to move the "After boot up, you will be prompted..." after the console output.